### DISCUSSION TITLE
【实际的情况】占位符填充模型不起作用

### DISCUSSION THREAD

**Discussion by mianma12138** [Tencent/libpag#2361] [Q&A] [answered]
占位符填充模型不起作用，设计这边输出四种模式，不起作用，开发代码更改也不起作用，请问这是什么情况。
![Uploading Snipaste_2024-07-08_11-32-25.png…]()
![Uploading 微信图片_20240708113348.png…]()

**Comment by mianma12138** [comment]
<img width="658" alt="微信图片_20240708113348" src="https://github.com/Tencent/libpag/assets/92774467/4332e508-3957-495e-ba30-10fc8d1d6bf2">

**Comment by mianma12138** [comment]
<img width="598" alt="Snipaste_2024-07-08_11-32-25" src="https://github.com/Tencent/libpag/assets/92774467/39b79602-4d95-4c1c-a1aa-22b35e9609d3">

**Comment by mianma12138** [comment]
修改为任何模式，都不起作用

**Comment by mianma12138** [comment, marked as answer]
已解决，设计侧无需设置填充方式，只需要开发单独设置某个占位符的填充方式即可。

**Reply by Mr-xue** [reply]
你好，我这边使用web端的setScaleMode设置填充模式也不生效，你说的设计侧无需设置填充方式是如何设置? 我刚看ae里面导出的设置项没有不设置啊

**Reply by kevingpqi123** [reply]
填充模式开发同学可以通过 SDK 的 API 接口设置， 覆盖 pag 文件中的填充模式

**Reply by ryan1051** [reply]
> 填充模式开发同学可以通过 SDK 的 API 接口设置， 覆盖 pag 文件中的填充模式

你好，Android下使用setScaleMode不生效是为什么呢，pagview和pagfile的大小不一致的情况
